### ISSUE TITLE
Powders

### ISSUE THREAD

**Issue by Alex-Braver** [Wynncraft/Issues#6646]
My powder sacs are not stacking properly. I've tried throwing them on the ground and picking them up and on different servers and after re-logging in. They are the same level and type of powder but not stacking.

**Comment by LordHelmchen98** [comment]
Are some of them powders you had before gavel? If so just upgrade them and it should work again.

**Comment by Alex-Braver** [comment]
So you're saying powders from Wynn Province and powders from Gavel Province don't stack? I can combine them and upgrade them, but I have to hold on to them to get the number I need to upgrade, which takes up so much room in my bank. Why not fix this stacking bug?

**Comment by LordHelmchen98** [comment]
No its not like powders from gavel and wynn dont stack but powders you got before gavel was released and powders you can get nowadays dont stack. The reason for that is that powders are just items with a name and its shown to you as a powder with specific information, but before gavel was released they had different names than now.
This is an explanation that was posted on the forums:

"Alright, let me explain you y'all why they don't stack and why we can't fix it.
So the way an online game works is with a client and a server. The client is you, the server is us. They talk to each other in order to make everything works. When a mob drops a powder, the server actually drops an inksac with the name FIRE 2. It then tells the client: "Yo, this inksac you see on the ground? It's actually a red dye." Same thing when you look at the item information: "Yo, if you see an element name with a number, just write all these powder info instead of just showing FIRE 2."

So the reason why they don't stack, is because you guys are mixing the old powder with a different name (Green Powder II) to the new ones (EARTH 2). The server goes: "WAIT A MINUTE CLIENT, YOU CAN'T STACK THOSE BECAUSE THEY AREN'T THE SAME ITEM AT ALL!". So unless we modify every old powder to the new names, it won't stack. It's kinda dangerous to modify items server-side, so we prefer changing it client-side only. You can however make them stack if you upgrade them."

So if some of your powders are powders you got before gavel was released in december last year, you have to upgrade them.
If they are powders you got since powders were released again with the gameplayupdate I dont know why they dont stack when you throw them on the ground. Normally if you throw them both on the ground next to each other and wait for a few seconds they should stick together and then be stackable.

**Comment by Alex-Braver** [comment]
I've been only playing for the last couple of months, so I don't think I have powders from before this update you're talking about.

**Comment by LordHelmchen98** [comment]
Hmm strange. But you can upgrade them, right? And does this happen to you often?

**Comment by Alex-Braver** [comment]
Yes if I include them with others in an upgrade it fixes the problem, but you see the problem, I have to hold 2 sets of almost every color and level until I get enough to upgrade. And it does happen often. It does seem like mob drops and gifted items are the difference. This started almost right away, for the last month. It's just more pronounced now as I play and pick up more powders. If I had more room in the bank I'd put up with this bug, but I'm already at bank max limit capacity, too bad it stops at level 50 for bank space growth.

**Comment by ColinRioux** [comment]
Please note that different **tier** powders don't stack. 

Also more specifically, the NBT information is different between old & new powders, which causes the issue. It's extremely hard and a waste of time to recall old powders, so the result is that they are un stackable.

**Comment by Rellia** [comment]
I have the exact same problem as this person. Sometimes I come across a single powder that has the exact same name and item ID as the others, it is exactly the same item, tier and stat values, absolutely everything is the same, but they will not ever stack with the normal ones. Usually these odd powders come from reward chests of some kind. Some can be fixed by throwing them on the ground and picking up after, but most of the time you just have to have 2 or three of the exact same powder in your inventory because they refuse to stack until you get one more and find a powder merchant.

**Comment by Alex-Braver** [comment]
But if I didn't play the game before until after the update I shouldn't be able to have any of these old powders, if that's the problem. I do know that someone threw item bombs the other day and managed to get some items from it. All of the powders I received from them would not stack with powders I already had in the bank, whether those were from mobs that I picked up or upgraded from the powder merchant. They were of the same tier and property but would not stack.

**Comment by Alex-Braver** [comment]
And it's good to know I'm not the only one having the problem. If there are others please comment.

**Comment by puddles31** [comment]
This problem is caused by getting powders from different sources. Powders gained from relics are not always stackable with powders gained from mob drops. Hope this clears this up.

**Comment by Alex-Braver** [comment]
What are relics - item bombs maybe? And does that mean this will never be fixed then? We're going to need a lot of room in our banks for 2 sets of powders until we can upgrade them to the next level? If this is the case, I put my vote in then for better banks in the forums to hold all these powders.

**Comment by puddles31** [comment]
Relics, or more accurately, Relic Shrines, are structures found throughout Wynn and Gavel. There's 15 in each province, and when you click on it, it will give you an item depending on the relic. E.g. There's a relic shrine next to the Lost Sanctuary dungeon, in the water. This is the relic you first find in the Relic Shrine quest. The relic gives you Water Powder I. Another example is Fire Powder I, which can be found in the Roots of Corruption (Nether) area. This probably won't be fixed, at least not for a while, but as for now bank you will probably get more powder from drops than relics. To make them stackable with other powders, upgrade them at a Powder Master and they will be stackable. Even if you don't, banks have more than enough room when fully upgraded.

**Comment by Alex-Braver** [comment]
Thank you for explaining that, that makes sense now. But they do take up a lot of space in my bank, which is fully upgraded, until I can upgrade the powders. Half of my bank is powders and the other half is quest items, and I'm having a very difficult time deciding which items I need to keep, as a lot of these items are needed for quests later on. If I had a double of every powder I picked up and was trying to get the sufficient number to upgrade them, they would take up my whole bank. Having 5 or 6 of these right now is a nuisance. And by what you are saying, everyone should be having this problem then. I just wish something gets fixed. For me, the easiest answer seems to be increase the bank, like make a page for powders, maybe I'm wrong.

**Comment by Moe-Ronickah** [comment]
The items all have NBT tags, and the situation is such that I have 3 of the same type and level of powder, and no two stack with each other.  It's being worked on, but don't hold your breathe, please.  Well, you can if you want, but you'll end up gasping for air before this can be fully resolved.

**Comment by Alex-Braver** [comment]
I'll be glad when it's fixed, but I think I'd still like to see another page in the bank for powders. Between them and quest objects there doesn't seem to be enough room in the bank as I progress through quests.

**Comment by Moe-Ronickah** [comment]
We now know why powders do not stack from drops, item bombs, the powder imps, and the relics - there are a couple of different resources that were accidentally used for the powders.  A fix is in the works, but we don't know when the fix will get completed and pushed to the game.  For now, you can save them and trade them up at the Powder Merchants.  All of the merchants give back the same kind of powders.

**Comment by DebugMC** [comment]
For anyone who does not know, dropping 2 of the same powder (that does not stack) will force them to stack. For example I have 2 fire 1 powders that do not stack, if i drop both on the same spot they stack.

**Comment by LordHelmchen98** [comment]
@DebugMC That works often but not everytime.

**Comment by Moe-Ronickah** [comment]
@DebugMC , please read my explanation as to what is currently the issue with powders.  Yes, some of the MINOR NBT data issues will change, if the powder NBT data is sourced from the tables.  HOWEVER, it makes not one bit of difference how many times you drop the powders if the have been sourced from different NBT data tables.  In this case, they are different powders, even though they look the same, and may have the same display name.  There is a major difference in their NBT tags and they will never stack with each other.

**Comment by SertSert5** [comment]
I understand the dilema now. Yay thanks friends :P

**Comment by Hidaro** [comment]
Same problem,i just had those two Powders from killing Trainers in the Temple of Legends,don't know why they don't stack,but they're still usable for upgrade ?

**Comment by DebugMC** [comment]
Yes

**Comment by Markles123** [comment]
If you need more space, make a new class, bring them to Ragni, and take things out of the bank and give them to your "junk" class, if you have a lot of classes available it really makes a wonderful way to store items
